pages/api/search/issues/index.ts

```diff
@@ -121,10 +130,17 @@ async function get(req: NextApiRequest, res: NextApiResponse) {
       currentPage: +paginatedData.page
     });
   } else {
+
+    const fields = sortBy?.reduce((prev, current, index) => {
```

Comment:
in either way, you can pull the logic up on the function and not have to repeat it